src/devinterp/slt/llc.py

```diff
@@ -6,106 +6,120 @@
         self.llc_per_chain = torch.zeros(num_chains, dtype=torch.float32).to(device)
-        self.llc_mean = torch.tensor(0., dtype=torch.float32).to(device)
-        self.llc_std = torch.tensor(0., dtype=torch.float32).to(device)
+        self.llc_mean = torch.tensor(0.0, dtype=torch.float32).to(device)
+        self.llc_std = torch.tensor(0.0, dtype=torch.float32).to(device)
 
         self.device = device
 
-    def update(self, chain: int, draw: int, loss: float):
-        self.losses[chain, draw] = loss 
-
-    @property
-    def init_loss(self):
-        return self.losses[0, 0]
+    def update(self, chain: int, draw: int, loss: float, init_loss):
+        self.losses[chain, draw] = loss
+        self.init_loss = init_loss  # This is clunky, sorry
 
     def finalize(self):
         avg_losses = self.losses.mean(axis=1)
-        self.llc_per_chain = (self.n / self.n.log()) * (avg_losses - self.init_loss)
+        self.llc_per_chain = self.temperature * (avg_losses - self.init_loss)
         self.llc_mean = self.llc_per_chain.mean()
         self.llc_std = self.llc_per_chain.std()
-        
+
     def sample(self):
         return {
             "llc/mean": self.llc_mean.cpu().numpy().item(),
             "llc/std": self.llc_std.cpu().numpy().item(),
-            **{f"llc-chain/{i}": self.llc_per_chain[i].cpu().numpy().item() for i in range(self.num_chains)},
+            **{
+                f"llc-chain/{i}": self.llc_per_chain[i].cpu().numpy().item()
+                for i in range(self.num_chains)
+            },
             "loss/trace": self.losses.cpu().numpy(),
         }
-    
-    def __call__(self, chain: int, draw: int, loss: float):
-        self.update(chain, draw, loss)
 
+    def __call__(self, chain: int, draw: int, loss: float, init_loss: float):
+        self.update(chain, draw, loss, init_loss)
 
 
 class OnlineLLCEstimator(SamplerCallback):
     """
-    Callback for estimating the Local Learning Coefficient (LLC) in an online fashion during a sampling process. 
+    Callback for estimating the Local Learning Coefficient (LLC) in an online fashion during a sampling process.
     It calculates LLCs using the same formula as LLCEstimator, but continuously and including means and std across draws (as opposed to just across chains).
 
     Attributes:
         num_chains (int): Number of chains to run. (should be identical to param passed to sample())
         num_draws (int): Number of samples to draw. (should be identical to param passed to sample())
-        n (int): Number of samples used to calculate the LLC.
+        temperature (float): Temperature used to calculate the LLC.
         device (Union[torch.device, str]): Device to perform computations on, e.g., 'cpu' or 'cuda'.
     """
-    def __init__(self, num_chains: int, num_draws: int, n: int, device="cpu"):
+
+    def __init__(
+        self, num_chains: int, num_draws: int, temperature: float, device="cpu"
+    ):
```

Comment:
just leaving this comment here, but it's a broader request: if there's going to be lots of files touched by both a refactor/feature update + linting, and if it's not too much trouble, can the linting be split out into a different PR? just so that it's easier to know which lines of change need what level of attention